packages/cli/tests/cli.test.ts

```diff
@@ -53,6 +56,39 @@ const testnetNetwork = new CLINetworkAdapter(
   {} as CLI_NETWORK_OPTS
 );
 
+describe('decode_cv', () => {
```

Comment:
A jest test for reading from stdin (e.g. `echo 0x1234 | stx decode_cv -`)  requires some trickier with mocking stdin, which doesn't seem worth it imo. It works when testing locally.

Reply:
> A jest test for reading from stdin (e.g. `echo 0x1234 | stx decode_cv -`)  requires some trickery with mocking stdin, which doesn't seem worth it imo. It works when testing locally.